src/openklant/tests/fixtures/migrate_command/cassettes/digitaal_adres.yaml

```diff
@@ -0,0 +1,47 @@
+interactions:
```

Comment:
I think I would put these cassettes under `src/openklants/tests/files/vcr_cassettes/...` instead of `/fixtures`

Reply:
I put these in the `migration/cassettes` folder, where there also is an fixture folder (for the V1 setup).